client/src/index.scss

```diff
@@ -0,0 +1,8 @@
+$threads-font-path: '~plaid-threads/fonts';
+@import '~plaid-threads/scss/typography';
+@import '~plaid-threads/scss/variables';
+
+body {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
threads was already imported in App.scss, and there are body styles there already as well. What is this file for? seems like a duplicate